src/Handlebars/Context.php

```diff
@@ -32,21 +33,33 @@ class Context
      */
     protected $index = [];
 
+    /**
+     * @var array dataStack stack for data within sections
+     */
+    protected $dataStack = [];
+
     /**
      * @var array key stack for objects
      */
     protected $key = [];
 
+    /**
+     * @var bool enableDataVariables true if @data variables should be used.
+     */
+    protected $enableDataVariables;
+
     /**
      * Mustache rendering Context constructor.
      *
      * @param mixed $context Default rendering context (default: null)
+     * @param bool $enableDataVariables Enables data variables (default: false)
      */
-    public function __construct($context = null)
+    public function __construct($context = null, $enableDataVariables = false)
```

Comment:
Trying to think ahead here, maybe this second parameter should be an array of options like you did over on the main class? That way if stuff crops up in the future we're not just endlessly tacking on parameters and maybe have less API breakage that way.

Reply:
Good idea - I've updated the code to reflect the suggestion.